### PR TITLE
Add Sidebar

### PR DESCRIPTION
**Pull request by dherault**
Not the definitive Sidebar but has enough elements to fit in an implementation draft, so it can then be enhanced with what's missing.

### CODE REVIEW

File: src/components/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+              {user.email}
+            </TransitionText>
+          </Box>
+        </Box>
+      </Box>
+    </Container>
+  )
+}
+
+export default Sidebar
```
Comment (michaeljguarino) [inline src/components/Sidebar.jsx:323]: Just realized this, but should we be using default exports?  Feel like that's going to make it more tricky to export everythign at the node module level

File: src/components/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+      const hasChildren = Array.isArray(items) && items.length > 0
+
+      return (
+        <Fragment key={id}>
+          <Item
+            id={activeId === id ? 'active-item' : ''}
+            active={activeId === id}
+            direction="row"
+            align="center"
+            width={`calc(100% - ${marginLeft}px)`}
```
Comment (michaeljguarino) [inline src/components/Sidebar.jsx:172]: why do we need this calc width?

File: src/components/Sidebar.jsx (hunk 1 above)
Comment (dherault) [reply src/components/Sidebar.jsx:323]: in index.js:  `export { default as Button } from './components/Button'`, and then anywhere:
`import { Button } from 'plural-design-system'`

Comment (michaeljguarino) [reply src/components/Sidebar.jsx:323]: yeah although if they were just standard exports, can't you do `export * from './components/Button'` which is a bit cleaner?  I seem to remember this is recommended for that reason

Comment (dherault) [reply src/components/Sidebar.jsx:323]: If you agree with [this](https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad) we may use named exports. I still prefer default but it's your call now :)

**Pull request merged**